extensions/netlify/package.json

```diff
@@ -5,6 +5,7 @@
   "description": "View your Netlify sites, domains, and team members",
   "icon": "icon.png",
   "author": "destiner",
+  "contributors": ["jonohewitt"],
```

Comment:
@jonohewitt we don't really set the contributors property in our extensions

Reply:
Okay no problem! Did I misunderstand something here or does it need updating?

https://github.com/raycast/extensions/blob/b946ec2ca121f9ab95dcf6850728ecb6380d01ba/basics/contribute-to-an-extension.md?plain=1#L23